modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -703,13 +704,32 @@ private void _addListTypeDefinition(ObjectDefinition objectDefinition)
 
 	private void _addObjectDefinitionResources(
 			Set<String> accountEntryRestrictedObjectRelationshipsNames,
-			ObjectAction[] objectActions, long objectDefinitionId,
-			ObjectLayout[] objectLayouts,
+			ObjectAction[] objectActions,
+			com.liferay.object.model.ObjectDefinition
+				serviceBuilderObjectDefinition,
+			long objectDefinitionId, ObjectLayout[] objectLayouts,
 			ObjectRelationship[] objectRelationships,
 			ObjectValidationRule[] objectValidationRules,
 			ObjectView[] objectViews)
 		throws Exception {
 
+		List<ObjectRelationship> relationships = ListUtil.fromArray(
+			objectRelationships);
```

Comment:
I think we can change the parameter type to `List<ObjectRelationship>`, and adapt this parameter when this method is called like:
```
 _addObjectDefinitionResources(...,
	ListUtil.fromArray(objectDefinition.getObjectRelationships()),
...);
```

This way we won't need to create an extra variable. What do you think?

Reply:
I had it done over here because i would have to change that in more than one call from outside the method, and i was being cautious about changing more than needed, but i agree with you. 
Will do that